docs/docs/reference/vue/_category_.json

```diff
@@ -0,0 +1,4 @@
+{
+  "label": "Vue 3",
```

Comment:
There's a jump between Vue 2 and Vue 3, and also lots of frustration from Vue 3 developers to see some vue-related libraries are not yet fully compatible with Vue 3 after almost 2 years. We could mention it in the intro though

Reply:
I know, but since Vue 3 is the latest version and that was the version that we were considering while preparing the SDK. Though I agree that we should mention it on the main page of the SDK docs, it doesn't make too much sense to me to emphasize on the version number in the sidebar. Especially that we use `v3` as a peer dependency, so it doesn't make too much of a difference.